docker/sink-compose.yml

```diff
@@ -17,6 +17,9 @@ services:
   sink-server:
     image: "hapiproject/hapi:latest"
     container_name: sink-server
+    environment:
```

Comment:
Thanks for fixing this but can we please separate this into another PR just for fixing issue #107? This seems to be unrelated to issue #194, right?

Reply:
yup, will remove this and add to a new PR!